src/components/RichTextEditor/utils/getMinWidthIfEmpty.ts

```diff
@@ -0,0 +1,19 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { CSSProperties } from "react";
+import { BaseEditor } from "slate";
+import { ReactEditor } from "slate-react";
+
+import { isEditorEmpty } from "@components/RichTextEditor/utils/isEditorEmpty";
+import { getCanvasFontSize, getTextWidth } from "@components/RichTextEditor/utils/getTextWidth";
+
+export const getMinWidthIfEmpty = (
+    editor: BaseEditor & ReactEditor,
+    placeholder: string,
+    wrapperEl: HTMLDivElement | null,
+): CSSProperties | undefined => {
+    if (isEditorEmpty(editor) && wrapperEl) {
```

Comment:
```suggestion
    if (wrapperEl && isEditorEmpty(editor)) {
```
for performance reasons